src/translations/en/about.json

```
@@ -0,0 +1,22 @@
+{
+  "about_heading": "About NVA",
+  "description": {
+    "heading": "Description",
+    "paragraph0": {
+      "intro": "With NVA, Norwegian research institutions get a service to:",
```

dette er direkte oversatt fra norsk eller? 😅

---

haha, sitate Hege: "Her kommer engelsk oversettelse (den kan gjerne fikses litt på):"
men jeg har brukt minimalt med tid til å fikse på det, men bare legg inn forslag hvis du ser noe som enkelt kan bli mye bedre 😅

---

Ok, ikke egentlig vår jobb å bruke tid på det, men...
```suggestion
      "intro": "By using NVA, Norwegian research institutions are able to:",
```